samples/angular-todo/src/environments/environment.ts

```diff
@@ -1,9 +1,9 @@
 export const environment = {
-  AM_URL: '',
-  REALM_PATH: '',
-  WEB_OAUTH_CLIENT: '',
-  JOURNEY_LOGIN: '',
-  JOURNEY_REGISTER: '',
-  API_URL: '',
-  production: '',
+  AM_URL: 'https://openam-ryan-bas.forgeblocks.com/am',
+  REALM_PATH: 'alpha',
+  WEB_OAUTH_CLIENT: 'example-react-app',
+  JOURNEY_LOGIN: 'Login',
+  JOURNEY_REGISTER: 'Registration',
+  API_URL: 'http://localhost:9443',
+  production: 'undefined',
```

Comment:
We probably don't want to commit this change.

Reply:
i _hate_ this file, i need to spend time to find a way to avoid doing this or untrack this file.